zebra/zebra_dplane.c

```diff
@@ -484,12 +484,13 @@ static void dplane_ctx_free(struct zebra_dplane_ctx **pctx)
 	case DPLANE_OP_NH_INSTALL:
 	case DPLANE_OP_NH_UPDATE:
 	case DPLANE_OP_NH_DELETE: {
-		if ((*pctx)->u.rinfo.nhe.ng.nexthop) {
-			/* This deals with recursive nexthops too */
-			nexthops_free((*pctx)->u.rinfo.nhe.ng.nexthop);
+		/* Labels may have been allocated */
+		nexthop_del_labels(&(*pctx)->u.rinfo.nhe.nexthop);
```

Comment:
There was a nullity check on `(*pctx)->u.rinfo.nhe.ng.nexthop` before, and `nexthop_del_labels` dereferences without one; do we need to preserve the nullity check?

Reply:
I think that _was_ a pointer, but now is an embedded object, so it can't be NULL.